Makefile-libs.Inc

```diff
@@ -55,17 +55,23 @@ POSTRS_SETUP_ZIP = libpost-$(platform)-v$(POSTRS_SETUP_REV).zip
 POSTRS_SETUP_URL_ZIP ?= https://github.com/spacemeshos/post-rs/releases/download/v$(POSTRS_SETUP_REV)/$(POSTRS_SETUP_ZIP)
 POSTRS_PROFILER_ZIP = profiler-$(platform)-v$(POSTRS_SETUP_REV).zip
 POSTRS_PROFILER_URL ?= https://github.com/spacemeshos/post-rs/releases/download/v$(POSTRS_SETUP_REV)/$(POSTRS_PROFILER_ZIP)
+POSTRS_SERVICE_ZIP = service-$(platform)-v0.5.0-alpha1.zip
+POSTRS_SERVICE_URL ?= https://github.com/spacemeshos/post-rs/releases/download/v0.5.0-alpha1/$(POSTRS_SERVICE_ZIP)
```

Comment:
TODO: update to not hardcode a version here, but use the general post-rs version